src/snyk/snykCode/utils/analysisUtils.ts

```diff
@@ -11,6 +11,7 @@ import {
   SNYK_SEVERITIES,
 } from '../constants/analysis';
 import { completeFileSuggestionType, ICodeSuggestion, ISnykCodeResult, openedTextEditorType } from '../interfaces';
+import { IssueUtils } from './issueUtils';
 
 export const createSnykSeveritiesMap = () => {
```

Comment:
minor: if you got time, maybe fix this linter message.

Reply:
done ✔️ There are few left but it'll creep the scope of the PR then.